prometheus-exporters/vrops-exporter/alerts/nsxt.alerts

```diff
@@ -0,0 +1,115 @@
+groups:
+- name: nsxt.alerts
+  rules:
+  - alert: NSXTDistributedFirewallSectionUsageLimitExceeded
+    expr: |
+        vrops_nsxt_sys_capacity_distributed_firewall_section_usage_count >
+        vrops_nsxt_sys_capacity_distributed_firewall_section_max_supported_count
+    for: 10m
+    labels:
+      severity: info
+      tier: vmware
+      service: compute
```

Comment:
Is this really `compute`? Or is there a `network` category?

Reply:
Same question applies for the following alerts.

Reply:
yes, it is an InfraOps topic, therefore we need it in the #alert-vmware-warning channel.

Reply:
(info first...)

Reply:
Alright.

Reply:
wait... routing is based on the tier. my bad.